signatures/gemss/gemss/param/256.yaml

```diff
@@ -0,0 +1,11 @@
+name: sign/GeMSS256
+security level:
+	classical: 256
+	quantum: 149
+	nist category: 5
+failure probability: -1.3306513097844322e+19
```

Comment:
we store this as logarithms, is the probability really 2^-133065130978443220000 ?

Reply:
yes, the expression is doubly exponential

Reply:
ok that's a new level of negligible :D